docker-rancher/azuredeploy.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
JSON formatting is off in the files. It'd be good to pass them all through a formatter